### PR TITLE
Implementing loading the client class from json

### PR DESCRIPTION
**Pull request by Tymotheus**

### CODE REVIEW

File: src/main/java/org/unice/model/Client.java [hunk 1]
```diff
@@ -1,60 +1,24 @@
...
 import lombok.*;
 import org.unice.model.submodel.Address;
 
 import java.util.List;
-import java.util.Map;
-import java.util.Optional;
 
 @Getter
 @Setter
 @Data
```
Comment (Tymotheus) [inline src/main/java/org/unice/model/Client.java]: Based on: https://projectlombok.org/features/Data
"@Data - A shortcut for @ToString, @EqualsAndHashCode, @Getter on all fields, @Setter on all non-final fields, and @RequiredArgsConstructor!"
I think only the @data decorator might be enough out of those 3 :)

Comment (damedomey) [reply src/main/java/org/unice/model/Client.java]: Ok, i'll correct it. Thanks

**Comment by damedomey** [comment]
your target branch is **main** 😔. That is why i create new one that target **dev**

**Comment by damedomey** [comment]
We can continue the discussion on the new PR

**Pull request closed without merging**